code/modules/mob/living/carbon/xenomorph/castes/carrier/abilities_carrier.dm

```diff
@@ -338,3 +338,88 @@ GLOBAL_LIST_INIT(hugger_images_list,  list(
 	turret.ammo = GLOB.ammo_list[GLOB.hugger_to_ammo[X.selected_hugger_type]]
 	succeed_activate()
 	add_cooldown()
+
+// ***************************************
+// *********** Call of Younger
+// ***************************************
+
+/datum/action/xeno_action/activable/call_younger
+	name = "Call of Younger"
+	action_icon_state = "call_younger"
+	mechanics_text = "Appeals to the larva inside the Marine. The Marine loses his balance, and larva's progress accelerates."
+	ability_name = "call younger"
+	plasma_cost = 150
+	cooldown_timer = 20 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_CALL_YOUNGER,
+	)
+
+
+/datum/action/xeno_action/activable/call_younger/can_use_ability(atom/A, silent, override_flags)
+	. = ..()
+	if(!.)
+		return
+
+	if(!ishuman(A))
+		if(!silent)
+			A.balloon_alert(owner, "Not human")
+		return FALSE
+
+	if(!(locate(/obj/item/alien_embryo) in A))
+		if(!silent)
+			A.balloon_alert(owner, "He's not infected")
+		return FALSE
+
+	var/mob/living/livingtarget = A
```

Comment:
Technically this is fine, but what I meant is that because you've done an ishuman check above, you should typecast A as human.

Reply:
Done?